src/settings.js

```diff
@@ -1758,7 +1758,7 @@ var MIN_IE_VERSION = 0x7FFFFFFF;
 // Edge 44.17763 was released on November 13, 2018
 // MAX_INT (0x7FFFFFFF, or -1) specifies that target is not supported.
 // [link]
-var MIN_EDGE_VERSION = 44;
+var MIN_EDGE_VERSION = 0x7FFFFFFF;
```

Comment:
Each of these directives have comment fields above them, specifying which browser versions released when correspond to the associated version.

Could you append a new comment (please keep the older version comment intact, so people have a good landmark point/hint to revert back to) for each comment that discusses the new version and its release date, and target min hardware that it enabled, etc. ?